CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 * Update real user metrics LUX.js to v300 ([PR #2637](https://github.com/alphagov/govuk_publishing_components/pull/2637))
 * Remove `header-navigation.js` ([PR #2632](https://github.com/alphagov/govuk_publishing_components/pull/2632))
+* Add support for Rails 7
+* Add support for Ruby 3.0 and 3.1
+* Drop support for Ruby 2.6 which is at End of Life (EOL)
```

Comment:
It'd be good if this could follow the existing convention:

```suggestion
* Add support for Rails 7; add support for Ruby 3.0 and 3.1; and drop support for Ruby 2.6 ([PR #2642](https://github.com/alphagov/govuk_publishing_components/pull/2642))
* ```

Reply:
Of course – thanks @injms